torch_geometric/nn/conv/gcn_conv.py

```diff
@@ -171,8 +177,10 @@ def __init__(
         improved: bool = False,
         cached: bool = False,
         add_self_loops: bool = True,
+        t: float = 1.0,
         normalize: bool = True,
-        bias: bool = True,
+        bias: bool = False,
```

Comment:
```suggestion
        bias: bool = True,
```
Keep it as`True` for compatibility.